src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Mappers/Apprentice/DetailsViewModelMapperTests.cs

```diff
@@ -177,7 +177,7 @@ public async Task ThenAllowEditApprenticeIsMappedCorrectly(ApprenticeshipStatus
             Assert.AreEqual(expectedAllowEditApprentice, _fixture.Result.AllowEditApprentice);
         }
 
-        [TestCase(DeliveryModel.PortableFlexiJob, false)]
+        [TestCase(DeliveryModel.PortableFlexiJob, true)]
```

Comment:
We can probably delete this test now, it was only there to ensure edit wasn't possible for Portable apprenticeships.

Reply:
Agree